tasks/Functions-client/request.js

```diff
@@ -18,7 +18,7 @@ task("functions-request", "Initiates a request from an Functions client contract
   .setAction(async (taskArgs, hre) => {
     // A manual gas limit is required as the gas limit estimated by Ethers is not always accurate
     const overrides = {
-      gasLimit: 500000,
+      gasLimit: 1500000,
```

Comment:
I remember we had a task argument that could override this. Can't we use a task argument and fall back to 50K?

Reply:
The gaslimit argument is for the amount of gas used for the callback transaction, not the transaction which initiates the request.

Reply:
However, I can add an optional `requestgas` task arg.

Reply:
You are right. Long term I believe we should always take hardhat.config.js value from the corresponding network{}, if specified. gasLimit is supposed to be very different for sepolia and mumbia and this will cause confusion for many people switching network.